### PR TITLE
Fix an error and a typo in the manual.

### PR DESCRIPTION
**Pull request by jasone**

### CODE REVIEW

File: doc/manual.wiki [hunk 1]
```diff
@@ -345,7 +345,7 @@ let log msg =
   in
   Lwt_io.printlf "%s: %s" thread_id msg
 
-lwt () =
+let lwt =
```
Comment (hcarty) [inline doc/manual.wiki]: I think this should be `let%lwt () = ...`

Comment (jasone) [reply doc/manual.wiki]: Cool, thanks for helping me learn the right way to do it along the way. :)

**Comment by aantron** [comment]
Thanks!

**Comment by aantron** [comment]
@jasone, off-topic, but I believe your fork is fork 100 of Lwt, and your PR triggered [build 1000](https://ci.appveyor.com/project/aantron/lwt/build/1000) in AppVeyor. You're the lucky winner! (No scam, I promise :p)

**Pull request merged**